internal/pkg/cgroups/cgroups.go

```diff
@@ -7,48 +7,68 @@ package cgroups
 
 import (
 	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
 
 	"github.com/containerd/cgroups"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
-const singularity = "/singularity"
+const (
+	singularity = "/singularity"
+)
 
 // Manager manage container cgroup resources restriction
 type Manager struct {
-	Name         string
-	Pid          int
-	parentCgroup cgroups.Cgroup
-	childCgroup  cgroups.Cgroup
+	Path   string
+	Pid    int
+	cgroup cgroups.Cgroup
+}
+
+// GetCgroupRootPath returns cgroup root path
+func (m *Manager) GetCgroupRootPath() string {
+	path := ""
```

Comment:
I think this var can be omitted, it is never set, and simple `return ""` is clear enough

Reply:
Done